spec/lib/user_importer_spec.rb

```diff
@@ -89,22 +89,18 @@
           end
         end
 
-        xit 'can import each row' do
-          expect(subject.import_by_row).to eq()
-        end
-
         it 'normalizes column names' do
-          actual = subject.normalize_column_names.first["Year"]
+          actual = subject.sanitized_rows.first["Year"]
           expect(actual).not_to eq(nil)
         end
       end
 
-      xit 'can import each row' do
-        expect(subject.import_by_row).to eq()
+      it 'imports things' do
+        subject.import_by_row
+        expect(User.all.count).to_not eq(0)
```

Comment:
Similar idea here, maybe we assert that the final user matches some characteristic or that we were able to import `User.all.count == 101` users (unsure how many we have).